src/activity/mod.rs

```diff
@@ -125,4 +131,11 @@ impl ActivityTaskManager {
         self.activity_heartbeat_manager_handle
             .record(details, t.div(2))
     }
+
+    pub(crate) fn is_known_not_found(&self, task_token: &TaskToken) -> bool {
+        self.outstanding_activity_tasks
+            .get(&task_token)
+            .map(|d| d.known_not_found)
+            .unwrap_or_default()
```

Comment:
This means return false if tt is not in the map right?

Reply:
Yep